salt/utils/logrotate.sls

```diff
@@ -0,0 +1,12 @@
+{% set logrotate_files = salt.pillar.get('logrotate') %}
+{% for name, settings in logrotate_files.items() %}
+
+generate_{{ name }}_logrotate:
+  file.managed:
+    - name: /etc/logrotate.d/{{ name }}
+    - source: salt://utils/logrotate/conf.jinja
+    - template: jinja
+    - mode: '0644'
+    - context:
+        logrotate_files: {{ logrotate_files }}
```

Comment:
I'm assuming this should be the `settings` variable from the `.items()` iteration